tests/unit_tests/test_wandb_artifacts_full.py

```diff
@@ -193,3 +194,29 @@ def make_table():
     artifact2.add(t1, "t2")
     assert artifact2.manifest.entries["t2.table.json"].ref is not None
     run.finish()
+
+
+def test_artifact_wait_success(wandb_init):
+    # Test artifact wait() timeout parameter
+    timeout = 2
```

Comment:
IIUC this timeout is supposed to be "large enough that the artifact-upload will always complete in less than this many seconds"; but 2sec is short enough that it's within the range of plausible execution times. If there's no countervailing reason to keep it small, could we make it Obviously Incredibly Big, like 60, or 3600, or 9999999?

(I'm partly worried about hiccups in CI scheduling causing this test to flake, and partly just trying to make the code as obvious as possible)